src/test/resources/db/scripts/schema.sql

```diff
@@ -5,6 +5,7 @@
         modified datetime(6),
         processed datetime(6),
         invoice_id varchar(255),
+        municipality_id varchar(255),
```

Comment:
Shouldn't it match the flyway script? (You have set the length to 4 in the flywayscript).

Maybe set the column length in the entity?
`@Column(name = "municipality_id", length = 4)`